doc/github_actions.rst

```diff
@@ -0,0 +1,35 @@
+Integration with GitHub Actions
+===============================
+
+`Universum` requires no special integration with `GitHub Actions <https://docs.github.com/en/actions>`_. It is usually
+launched as one long step in a single build stage.
+
+.. warning::
+
+    GitHub Actions doesn't support `report_artifacts` key in configuration.
+
+Command line
+------------
+
+To use `universum` to check-out current repository (the repository that contains the GitHub Action workflow) is
+necessary to use `GitHub Environment variables
+<https://docs.github.com/en/actions/learn-github-actions/environment-variables>`_:
+``python -m universum --vcs-type=git --git-repo $GITHUB_SERVER_URL/$GITHUB_REPOSITORY --git-refspec $GITHUB_REF_NAME``
+
+`Universum` also supports reports to review from GitHub Actions. Link to build are created using environment variables
```

Comment:
I propose to mention that reports to review only make sense for swarm, gerrit and github application (weird). Also it would be nice to mention that for GitHub Action triggered by GitHub pull request, the report to review is not needed, because GitHub already links them together.

Reply:
Well, if there is a check run, it should be possible to use it for posting to review. This is currently implemented for GitHub App, but the most of the code should be the same for the GitHub Action.
Anyway, it should be in separate commit.